### ISSUE TITLE
No option for remote login  (version 0.8)

### ISSUE THREAD

**Issue by ebojali** [openhardwaremonitor/openhardwaremonitor#1100]
Hi, 
I don't hav any option for remote login, I tried all version on your webpage.

Thanks
![snip20180413_5](https://user-images.githubusercontent.com/35561346/38756497-05ad938a-3f6a-11e8-9d4f-1287d04dea35.png)

**Comment by DarthDestroyer** [comment]
What do you mean with "remote Login"?

Are you using a RDP Session?

**Comment by ebojali** [comment]
yes I am but I am also connected via monitor locally. But I want to use the rig monitoring app on andriod as I've always done but I have to enable that option first before I can use the app on my phone.

**Comment by ebojali** [comment]
Anyone supporting this? :)

**Comment by DarthDestroyer** [comment]
if i would understand you, maybe...
i dont really know, what you want!

I give you some tips. 

1. if you are missing sensors or other Data, check if you had started as admin and after that, create a report and upload it in your issue!
2. the latest version is only available at github. 
3. rdp is and will ever be a problem! there are several issues! but i think, that it isnt relevant here. 

if you could upload a Report, that would help! 
It is somewhere at File>Report>Create/Save dont send it! Attach it here, please!

**Comment by ebojali** [comment]
As mentioned in the installation instructions on this webpage:
http://rigmonitoring.net/home?locale=en

I did not get any RIG identifier and do not have the option to enable remote log.. I noticed that I wrote "remote login", sorry about that. I ment remote log of course.

If I downloaded the application linked from the webpage I got everything working, but not the version on github.

**Comment by DarthDestroyer** [comment]
i think that you have to search in that application and not in OHM!

i dont know it, but you have to search there and not in ohm, maybe. 
that thing uses the library and does whatever it does with it! 

and if you download the zip from github, it is only the ohm Sourcecode! You have to compile it before you can use it... 

had you searched for help, at there forum / github or whatever they use?
